### PR TITLE
SEED-714: Template menu

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp can you please review :)

File: iogt/static/css/_base.scss [hunk 1]
```diff
@@ -1,11 +1,10 @@
-
 html, body {
   margin: 0;
   background: #fff;
 }
 
 html, body, input, textarea {
-  font-family: Arial, Helvetica, sans-serif;
+  font-family: 'Open Sans', Arial, Helvetica, sans-serif;
```
Comment (peterp) [inline iogt/static/css/_base.scss]: We can't use webfonts in feature phones.

File: iogt/static/css/_base.scss [hunk 2]
```diff
@@ -29,3 +34,16 @@ h1, h2, h3, h4, h5, h6 {
   font-size: 16px;
```
Comment (peterp) [inline iogt/static/css/_base.scss]: Why do we have font settings in an `img` tag? Does this affect the `alt` property?

File: iogt/static/css/variables/_colors.scss [hunk 3]
```diff
@@ -7,13 +7,21 @@ $roman          : #df6859;
...
-  mandy          : $mandy,
-  wewak          : $wewak,
-  robin-egg-blue : $robin-egg-blue,
-  roman          : $roman,
-  danube         : $danube,
-  dove-gray      : $dove-gray
+  pelorous        : $pelorous,
+  eucalyptus      : $eucalyptus,
+  saffron         : $saffron,
+  sandy_brown     : $sandy-brown,
```
Comment (peterp) [inline iogt/static/css/variables/_colors.scss]: why are we sometimes using an underscore `_` and other times a dash `-`? Why did you rename the colours?

File: iogt/static/css/variables/_colors.scss [hunk 4]
```diff
@@ -7,13 +7,21 @@ $roman          : #df6859;
 $danube         : #6093cd;
 $dove-gray      : #737373;
 
+$pelorous       : #37BFBE;
+$eucalyptus     : #2A9B58;
+$saffron        : #F2B438;
+$sandy-brown    : #EF9955;
+$cinnabar       : #EC3B3A;
+$medium-violet  : #B62A99;
+$indigo_blue    : #5F7AC9;
```
Comment (peterp) [inline iogt/static/css/variables/_colors.scss]: This has an underscore, other files have a dash?

File: iogt/templates/base.html [hunk 5]
```diff
@@ -21,6 +21,7 @@
         <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```
Comment (peterp) [inline iogt/templates/base.html]: We can't include this here, we need to use an import statement in `_STATE.scss` to include web-fonts.

File: iogt/templates/core/tags/footerpage.html [hunk 6]
```diff
@@ -2,9 +2,13 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% if footers %}
-  <ul class="footer__links">
+  <ul class="menu-list">
     {% for page in footers %}
-      <li><a href="{% pageurl page %}">{{page.title}}</a></li>
+      <li class="menu-list__item">
+      <a href="{% pageurl page %}">
```
Comment (peterp) [inline iogt/templates/core/tags/footerpage.html]: This indentation needs to be fixed.

**Comment by peterp** [comment]
@Mitso can you add screenshots?

File: iogt/static/css/layout/_l-footer.scss [hunk 7]
```diff
@@ -0,0 +1,10 @@
+.footer {
+  background: #f0f0f0;
+  color: #7B8C8F;
+  border-top: 2px solid #07292F;
+  font-size: 12px;
```
Comment (peterp) [inline iogt/static/css/layout/_l-footer.scss]: font sizes should be variables.

File: iogt/static/css/modules/_m-navigation.scss [hunk 8]
```diff
@@ -1,22 +1,25 @@
+.menu-list__item  {
```
Comment (peterp) [inline iogt/static/css/modules/_m-navigation.scss]: the file is called `_m-navigation.scss` but the CSS class is called `menu`, perhaps we should rename the file?

File: iogt/static/css/modules/_m-navigation.scss [hunk 9]
```diff
@@ -1,22 +1,25 @@
...
+  border-bottom: 1px solid #B0B9BB;
+  color: #07292F;
+  font-size: 12px;
```
Comment (peterp) [inline iogt/static/css/modules/_m-navigation.scss]: font sizes should be variables.

File: iogt/templates/base.html [hunk 10]
```diff
@@ -66,11 +67,12 @@
       {% endblock %}
 
       {% block footer %}
-      <div class="footer l-indent">
-        <div>&copy; The Internet of Good Things</div>
+      <div class="footer">
           {% footer_page %}
+          <div class="copyright l-indent">
```
Comment (peterp) [inline iogt/templates/base.html]: please place this outside the footer.

File: iogt/templates/core/tags/latest_listing_homepage.html [hunk 11]
```diff
@@ -8,8 +8,9 @@ <h2 class="heading l-indent--deep">{% trans "Recommended" %}</h2>
 {% for article in articles %}
   <li class="article-list__item">
     {% if article.image %}
-    <a href="{% pageurl article %}">{% image article.image width-80 class="article-list__image"%}</a>
+      <a href="{% pageurl article %}">{% image article.image width-80 class="article-list__image"%}</a>
```
Comment (peterp) [inline iogt/templates/core/tags/latest_listing_homepage.html]: Was this changed by accident?

File: iogt/static/css/modules/_m-navigation.scss [hunk 12]
```diff
@@ -19,7 +19,7 @@
  
 .menu-list__item--title {
   display: block; 
-  font-size: 14px;
+  font-size: $font-middle;
```
Comment (peterp) [inline iogt/static/css/modules/_m-navigation.scss]: `medium` may be a better word, based off of standard CSS sizes:

font-size: xx-small;
font-size: x-small;
font-size: small;
font-size: medium;
font-size: large;
font-size: x-large;
font-size: xx-large;

Is medium a good size here? Since it's smaller than the standard of 16px?

File: iogt/static/css/modules/_m-search.scss [hunk 13]
```diff
@@ -1,4 +1,17 @@
 
 .search {
-  text-align: center;
+  background: #f0f0f0;
+  position: relative;
+  padding: 15px;
+
+  input[type="submit"] {
```
Comment (peterp) [inline iogt/static/css/modules/_m-search.scss:11]: There are a lot of magic numbers here that's magic the code smell; check out the work I did on Tune Me that's a bit cleaner (uses flexbox): https://github.com/praekelt/molo-tuneme/blob/develop/tuneme/static/new/css/state/_s-header.scss#L6

A lot of this isn't really necessary though... because we're just waiting for the feature phone templates. Maybe just stick to the HTML for now?

**Comment by Mitso** [comment]
I don't know the font used on the designs but looks something close to Open Sans to me. 
There's a lot of variation in font, font-weight which affects padding I think.

Footer Links 
footer-navigation-local

Section links and search form
navigation-menu-local

**Comment by peterp** [comment]
👍
